Shared/Extensions/Selectable.swift

```diff
@@ -146,28 +146,60 @@ class PanGestureRecognizer: UIPanGestureRecognizer {
     }
 }
 
+/// Similar to pan gesture recognizer except that the begin state is triggered after a short delay, even if the touch doesn't move.
```

Comment:
Now we don't immediately start the pan gesture for swiping up a message. This helps reduce conflicts with double tap gestures.